hermes/probe/read/read.go

```diff
@@ -106,19 +111,20 @@ func ReadFile(ctx context.Context, target *probe.Target, fileID int32, fileSize
 		default:
 			status = metrics.ProbeFailed
 		}
-		target.LatencyMetrics.APICallLatency[metrics.APIGetFile][status].Metric("hermes_api_latency_s").AddFloat64(time.Now().Sub(start).Seconds())
-		return fmt.Errorf("readFile(id: %d).%v: could not read file %s: %w", fileID, status, fileName, err)
+		target.LatencyMetrics.APICallLatency[metrics.APIGetFile][status].Metric(hermesAPILatencySeconds).AddFloat64(time.Now().Sub(start).Seconds())
+		return fmt.Errorf(".%q: could not read file %q: %w", status, fileName, err)
```

Comment:
Nit: I wouldn't use %q to format the status. Also, stray dot?

Reply:
changed to %v